flask_excel/__init__.py

```diff
@@ -18,20 +18,37 @@ def get_file_tuple(self, field_name):
         filename = filehandle.filename
         extension = filename.split(".")[1]
         return extension, filehandle
-    
 
 Flask.request_class = ExcelRequest
 webio.ExcelResponse = Response
 
-from pyexcel_webio import (
-    make_response,
-    make_response_from_array,
-    make_response_from_dict,
-    make_response_from_records,
-    make_response_from_book_dict,
-    make_response_from_a_table,
-    make_response_from_query_sets,
-    make_response_from_tables
-)
-
-__VERSION__ = '0.0.3'
+def add_file_name(response, file_name, file_type):
```

Comment:
These code changes look a bit lengthy. There must be a simpler way to do this. Please allow me to re-factor it along aside with pyexcel-webio.